contrib/ansible/roles/kernel/tasks/kernel_clean.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+- name: Delete kernel sources
+  shell: \rm -Rf /home/vagrant/net-next
```

Comment:
Ok i will see how I can make /home/vagrant configurable. Everything else is relative to it.